queries/synths/useFeeReclaimPeriodsQuery.ts

```diff
@@ -0,0 +1,82 @@
+import { Provider, Contract } from 'ethcall';
+import { ethers } from 'ethers';
+import { useQuery, UseQueryOptions } from 'react-query';
+import { CurrencyKey } from '@synthetixio/contracts-interface';
+import { wei } from '@synthetixio/wei';
+
+import Connector from 'containers/Connector';
+import ExchangerABI from 'lib/abis/Exchanger.json';
```

Comment:
Is there reason we're hand loading an ABI here? You should be able to get the Exchanger contract object from `@synthetixio/contracts-interface`.